### ISSUE TITLE
InternVL2.5-MPO的mathvista评估是使用internvl_chat/evaluate.sh，还是使用VLMEvalKit？

### ISSUE THREAD

**Issue by linhaojia13** [OpenGVLab/InternVL#887]
[InternVL2的文档](https://internvl.readthedocs.io/en/latest/internvl2.0/preference_optimization.html)指出InternVL2.0-MPO使用internvl_chat/evaluate.sh评估mathvista，但是[InternVL2.5的文档](https://internvl.readthedocs.io/en/latest/internvl2.5/preference_optimization.html)只提到了VLMEvalKit:
```
# Evaluation
We evaluate the performance on other benchmarks (e.g., MMVet, LLaVABench, and CRPE) using [VLMEvalKit](https://github.com/open-compass/VLMEvalKit). You need to set use_mpo_prompt=True in [config.py](https://github.com/open-compass/VLMEvalKit/blob/main/vlmeval/config.py) and USE_COT="1" in environment variable to activate the CoT prompt.
```

假如我要评估InternVL2.5-MPO的MathVista结果，应该使用internvl_chat/evaluate.sh还是VLMEvalKit?

**Comment by yuecao0119** [comment]
你好，

如我们`internvl_chat/eval/mathvista`的[README.md](https://github.com/OpenGVLab/InternVL/tree/main/internvl_chat/eval/mathvista#-overview)中所说，我们报告中的结果通过VLMEvalKit获得。
```markdown
For scoring, we use GPT-4-0613 as the evaluation model. While the provided code can run the benchmark, we recommend using [VLMEvalKit](https://github.com/open-compass/VLMEvalKit) for testing this benchmark if you aim to align results with our technical report.
```

**Comment by amoreZgx1n** [comment]
请问你在MPO过程中遇到过这个问题吗
{'loss': 0.0, 'learning_rate': 1.0000000000000001e-07, 'rewards/chosen': 0.0, 'rewards/rejected': 0.0, 'rewards/accuracies': 0.0, 'rewards/margins': 0.0, 'logps/rejected': 0.0, 'logps/chosen': 0.0, 'logits/rejected': 6.343141555786133, 'logits/chosen': 5.391749858856201, 'nll_loss': nan, 'epoch': 0.0}

**Comment by linhaojia13** [comment]
> 你好，
> 
> 如我们`internvl_chat/eval/mathvista`的[README.md](https://github.com/OpenGVLab/InternVL/tree/main/internvl_chat/eval/mathvista#-overview)中所说，我们报告中的结果通过VLMEvalKit获得。
> 
> For scoring, we use GPT-4-0613 as the evaluation model. While the provided code can run the benchmark, we recommend using [VLMEvalKit](https://github.com/open-compass/VLMEvalKit) for testing this benchmark if you aim to align results with our technical report.

Thank you very much!
